system/mp/mpcomm.cpp

```diff
@@ -1593,36 +1593,32 @@ class CMPPacketReader: public ISocketSelectNotify, public CInterface
 };
 
 
-CMPChannel::CMPChannel(CMPServer *_parent,SocketEndpoint &_remoteep) 
+CMPChannel::CMPChannel(CMPServer *_parent,SocketEndpoint &_remoteep) : channelsock(nullptr),
+            parent(_parent), remoteep(_remoteep), multitag(TAG_NULL), closed(false),
+            master(false), sendwaiting(0), attachaddrval(0)
```

Comment:
No need for these ctor initializers as well as member initializers, i.e. channelsock, multitag, closed, master, sendwaiting and attachaddrval